src/js/player.js

```diff
@@ -1939,7 +1942,6 @@ class Player extends Component {
       if (this.usingNativeControls_ !== bool) {
```

Comment:
Personally, I would invert this conditional and return early to get rid of a level of nesting in this function. Not worth blocking this PR, though.